diagnostics-app/tsconfig.json

```diff
@@ -3,11 +3,6 @@
   "compileOnSave": false,
   "compilerOptions": {
     "baseUrl": "./",
```

Comment:
`"baseUrl": "./src"`?

Reply:
Done.

All references will be defaulted from src.